### DISCUSSION TITLE
Xcode support

### DISCUSSION THREAD

**Discussion by deadsoul44** [community/community#38845] [Codespaces]
### Select Topic Area

Product Feedback

### Body

If you provide xcode support, that will be game changer. I will pay for it happily because I dont want to invest in a Mac. It will be like shut up and take my money :)

**Comment by jfullstackdev** [comment]
Xcode is a copyright of Apple, so GitHub can't provide support of that because they have no right to do so

**Comment by deadsoul44** [comment]
Mac server with xcode installed can be provided. There are companies already doing that like macincloud.

**Comment by matthewruzzi** [comment]
Are there any plans for supporting macOS-based CodeSpaces environments?
https://github.com/orgs/community/discussions/36604

Request: Windows and macOS images
https://github.com/orgs/community/discussions/151996
